idea-plugin/src/main/java/com/palantir/gradle/jdks/GradleJdksProjectService.java

```diff
@@ -90,7 +110,28 @@ public void maybeSetupGradleJdks() {
                     "Skipping setupGradleJdks because gradle JDK setup is not found %s", gradleSetupScript));
             return;
         }
-        setupGradleJdks();
+        TasksKt.withBackgroundProgress(
+                project,
+                "Gradle JDK Setup",
+                (_coroutineScope, continuation) -> {
+                    StepsKt.withProgressText(
+                            "`Gradle JDK Setup` is running. Logs in the `Gradle JDK Setup` window ...",
+                            (_cor, conti) -> {
+                                setupGradleJdks();
+                                return conti;
+                            },
+                            continuation);
+                    return continuation;
+                },
+                new Continuation<>() {
+                    @Override
+                    public @NotNull CoroutineContext getContext() {
+                        return EmptyCoroutineContext.INSTANCE;
+                    }
+
+                    @Override
+                    public void resumeWith(@NotNull Object _object) {}
+                });
```

Comment:
the Gradle JDK setup process would look similar to the Gradle Sync background process. The logs can be seen in `Gradle JDK Setup` window tab. 

<img width="1725" alt="Screenshot 2024-09-06 at 12 21 50 PM" src="https://github.com/user-attachments/assets/44e230f9-48c0-459e-9157-a1e15b1b18b1">